great_expectations/core/expectation_suite.py

```diff
@@ -174,8 +184,30 @@ def add_expectation(self, expectation: Expectation) -> Expectation:
                     raise exc
 
         expectation.register_save_callback(save_callback=self._save_expectation)
+
+        self._submit_expectation_created_event(expectation=expectation)
+
         return expectation
 
+    def _submit_expectation_created_event(self, expectation: Expectation) -> None:
+        if expectation.__module__.startswith("great_expectations."):
+            custom_exp_type = False
+            expectation_type = expectation.expectation_type
+        else:
+            custom_exp_type = True
+            anonymizer = Anonymizer()
+            expectation_type = anonymizer.anonymize(expectation.expectation_type)
```

Comment:
we're expecting expectation type to contain sensitive data?

Reply:
If its a custom name, we don't want to send it over the wire

Reply:
Having all this logic here feels a little bad though?

Reply:
It does seem like non-business logic in an area I'd want business logic only, but I'm not sure we have architecture that allows for anything better decoupled atm. I wouldn't block this PR on that being there.

Reply:
I'm wondering if we should wait to send these events until we are connected to a data context. Not a blocker.